src/Utils/kernel_launching.jl

```diff
@@ -94,10 +117,10 @@ function launch!(arch, grid, workspec, kernel!, kernel_args...;
                  only_active_cells = nothing,
                  kwargs...)
 
-    workgroup, worksize = work_layout(grid, worktuple(workspec);
-                                      include_right_boundaries,
-                                      reduced_dimensions,
-                                      location)
+    workgroup, worksize = work_layout(grid, workspec;
+                                     include_right_boundaries,
+                                     reduced_dimensions,
+                                     location)
```

Comment:
```suggestion
                                      include_right_boundaries,
                                      reduced_dimensions,
                                      location)
```

seems that the alignment went bit off here?